android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/BackButton.kt

```diff
@@ -0,0 +1,44 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Color.Companion.Transparent
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import vn.luongvo.kmm.survey.android.R
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+
+private const val Rotate180 = 180f
+
+@Composable
+fun BackButton(
+    modifier: Modifier,
+    onClick: () -> Unit
+) {
+    Button(
+        onClick = onClick,
+        modifier = modifier
+            .wrapContentSize()
+            .rotate(Rotate180), // switch Right Arrow to Back button
```

Comment:
Why don't we use the left arrow so we don't have to rotate on every buttons?

Reply:
I guess @luongvo wanna save a resource by reusing back button 😄

Reply:
Correct @doannimble @blyscuit, saving resources purpose 😛

Reply:
Right arrow in the design, appears only on the Survey screen (once), but left arrow appears in majority of the screens. We can reuse the left arrow by rotating once instead of rotating most of the time to save processing resource as well? 🤔

Reply:
@blyscuit In my IC, the right arrow appears in the Home and Survey Detail screen while the left arrow appears only in the Survey Detail screen. `Forgot password` process is not a required feature and my IC does not include it, just FYI.